src/client.js

```diff
@@ -182,6 +183,7 @@ Client.prototype.connect = function(args) {
 			} else {
 				webirc = {
 					password: config.webirc[network.host],
+					username: network.username,
```

Comment:
@dgw: We could, but it needs to be user changeable in case a server provides a username *and* password. So far the ircd I use for testing doesn't even use the username, but since it's in the spec we should implement it right.